typings/index.d.ts

```diff
@@ -510,15 +533,9 @@ export class DiscordBotsCo extends ServiceBase {
 /**
  * Represents the Discord Bots service.
  * @param token - The token/key for the service
- * @param options - The options of the service. Providing this is highly recommended.
- * @param options.library - The bot's library
- * @param options.clientID - The bot ID for the user agent
  */
 export class DiscordBotsGG extends ServiceBase {
-    constructor(token: string, options: {
-        library: string;
-        clientID: IDResolvable;
-    });
+    constructor(token: string);
```

Comment:
Why did this change?

Reply:
I have no idea of what happened, I'll run the generator again to see what happens